pages/chapter/[chapterNumber].js

```diff
@@ -19,38 +90,24 @@ export default function Chapter() {
 
       <div className='max-w-5xl font-inter py-24 chapter-intro mx-auto text-center  px-4 sm:px-6'>
         <img
-          src='arrow-left.png'
+          src='/arrow-left.png'
```

Comment:
What if we add links to these and make these functional instead of taking this forward

Reply:
Will do in another PR